src/app/Middleware/version.cs

```diff
@@ -46,7 +46,7 @@ public static IApplicationBuilder UseVersion(this IApplicationBuilder builder)
                         }
 
                         // build and cache the json string
-                        string json = "{ " + $"\"apiVersion\": \"{swaggerVersion}\", \"appVersion\": \"{Middleware.VersionExtensions.Version}\"" + " }";
+                        string json = "{ " + $"\"appVersion\": \"{Middleware.VersionExtensions.Version}\", \"apiVersion\": \"{swaggerVersion}\"" + " }";
```

Comment:
the order seems inverted to me - apiVersion sorts above appVersion and apiVersion is common across all languages whereas appVersion isn't.  What's driving the change?

Reply:
java and typescript repos are set to app and then api, so just consistency

Reply:
cancelling PR and making change to java instead